src/test/jtx/impl/AMMTest.cpp

```diff
@@ -103,44 +103,45 @@ AMMTestBase::testAMM(
     std::optional<std::pair<STAmount, STAmount>> const& pool,
     std::uint16_t tfee,
     std::optional<jtx::ter> const& ter,
-    std::optional<FeatureBitset> const& features)
+    std::vector<FeatureBitset> const& vfeatures)
```

Comment:
I don't love this interface - running the same test with a different set of features seems like you'd want another function call (with possibly another `ter`). I'm not that picky about test code, so no change needed if you prefer this.